packages/forma-36-website/src/pages/guidelines/copy/date-and-time/index.mdx

```diff
@@ -0,0 +1,239 @@
+---
+title: 'Date and time'
+---
+
+import {
+  Table,
+  TableHead,
+  TableBody,
+  TableCell,
+  TableRow,
+  Tooltip,
+} from '@contentful/forma-36-react-components';
+
+# Date and time
+
+## Absolute vs relative
+
+Consider the context when having to choose between absolute and relative date and time.
+
+<Table className="f36-margin-bottom--xl" style={{ width: '100%' }}>
+  <TableHead>
+    <TableRow>
+      <TableCell style={{ width: '25%' }}>Type</TableCell>
+      <TableCell style={{ width: '25%' }}>Example</TableCell>
+      <TableCell style={{ width: '50%' }}></TableCell>
+    </TableRow>
+  </TableHead>
+  <TableBody>
+    <TableRow>
+      <TableCell>Absolute</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>13 Aug 2019 at 10:00 am</TableCell>
+      <TableCell>
+        When is an entry scheduled to be published?
+        <br />
+        Relative time wouldn't be specific enough.
+      </TableCell>
+    </TableRow>
+    <TableRow>
+      <TableCell>Relative</TableCell>
+      <TableCell style={{ fontWeight: 700 }}>
+        <Tooltip content="13 Aug 2019 at 10:00 am">3 days ago</Tooltip>
+      </TableCell>
+      <TableCell>
+        When was my collaborator last active?
+        <br />
+        An absolute date would be unnecessarily specific, but do your best to
+        provide the absolute format if the user interacts with it.
```

Comment:
I think this might need some clarification. `but do your best to
        provide the absolute format if the user interacts with the content associated with the date.` maybe?

Reply:
I agree. How about:
`Absolute date and time would be unnecessarily specific. Provide the absolute date and time on user interaction, e.g. when the user hovers over relative time with their cursor.`?

Reply:
Ah cool, I get it now. Yeah, that's a lot better.